api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
+
+class UserProfile(models.Model):
+	user = models.OneToOneField(User) 
+	#id = models.AutoField(primary_key=True) 
+	fullname = models.CharField(max_length=100, blank=False)
+	githubuser = models.CharField(max_length=100, null=False, blank=False)
+	email = models.EmailField(blank=False)
```

Comment:
We did not define this field in our initial model https://docs.google.com/a/sustic.com/document/d/1KVQuXIsrgCxNmUrBOiThwbbWpJ6CFs4q-rGh7X5Vgiw/edit. Great idea though :smile: